typst.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "typst-jp-conf-template"
+version = "0.0.2"
+compiler = "0.12.0"
+entrypoint = "libs/mscs/lib.typ"
+repository = "https://github.com/kimushun1101/typst-jp-conf-template"
+authors = ["Shunsuke Kimura <https://sites.google.com/view/kimushun>"]
```

本件に対する対応は基本的にGitHubで行うことになるかと思いますので、リンク先はGitHubにしようと思います。
このプルリクエストだけでなく以前もプルリクエスト #1 を頂いているため、@m-tsuru さんも authors に加えようと考えているのですがいかがでしょうか？クレジット的な意味合いです。あくまで主著者は私で、責任などを分担させるものではありません（そもそもMIT-0ライセンス）。

```suggestion
authors = ["Shunsuke Kimura <https://github.com/kimushun1101>", "ささくらり <https://github.com/m-tsuru>"]
```

---

ありがとうございます。リンクの件の、私の件について、適用させていただきます。